lib/src/database.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:postgres/postgres.dart';
+
+class database {
+  Future<List<List<dynamic>>> signIn(String command) async {
+    var connection = PostgreSQLConnection(
+        "ec2-54-224-194-214.compute-1.amazonaws.com", 5432, "d1lanfsa3i6j9o",
+        username: "vszmgrceclzrsv",
+        password:
+            "52c26c636820b257d60e305c7fd431d35365368b1c49b8c092834cb494fc36d3",
+        useSSL: true);
```

Comment:
This is a really big security risk, just having the values hardcoded is not ideal. Look into using a `.env` file.